jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/resource/FileSystemPool.java

```diff
@@ -257,6 +257,19 @@ private void retain(URI fsUri, FileSystem fileSystem, Mount mount)
         }
     }
 
+    /**
+     * Used to test the reference count value for a URI during unit testing.
+     * @param fsUri the filesystem URI to fetch
+     * @return the reference count on that URI
+     */
+    protected int getReferenceCount(URI fsUri)
```

Comment:
I would make this method package-protected; we don't want anyone (including ourselves) to use it for anything but unit tests. And explicitly say in the javadoc something like "only used for testing".

Reply:
Done